winstore-template.csv

```diff
@@ -0,0 +1,430 @@
+﻿Field,ID,Type (Type),default,en-gb
+Description,2,Text,,"$STR_ABSTRACT$
+
+$STR_OVERVIEW_BUILD_TITLE$
+
+$STR_OVERVIEW_BUILD_TEXT$
+
+$STR_OVERVIEW_TRANSPORT_TITLE$
+
+$STR_OVERVIEW_TRANSPORT_TEXT$
+
+$STR_OVERVIEW_UPGRADE_TITLE$
+
+$STR_OVERVIEW_UPGRADE_TEXT$
+
+$STR_OVERVIEW_EXTEND_TITLE$
+
+$STR_OVERVIEW_EXTEND_TEXT$
+
+$STR_OVERVIEW_LICENSE_TITLE$
+
+$STR_OVERVIEW_LICENSE_TEXT$"
+ReleaseNotes,3,Text,,"OpenTTD 14.1 fixes a multiplayer desync bug and cleans up a few rough edges with the new ship pathfinder and the unbunching feature, plus plenty of other bugfixes. OpenTTD 14.0 introduces over 40 new features, over 500 bug fixes and 200 other changes, including:
+
+- New timekeeping, use real-time units if you want!
+- Improved ship pathfinding
+- Road vehicle unbunching
+- Social Platform integration
+- Survey support
+- GUI improvements, scalable font
+- New Help and Manuals option, improved default settings
+
+Check out our blogs on openttd.org for full details."
+Title,4,Text,,OpenTTD (official)
+ShortTitle,5,Text,,OpenTTD
+SortTitle,6,Text,,
+VoiceTitle,7,Text,,
+ShortDescription,8,Text,,$STR_ABSTRACT$
+DevStudio,9,Text,,OpenTTD Developers
+CopyrightTrademarkInformation,12,Text,,Copyright © OpenTTD Developers 2002-2024
```

Comment:
Maybe automate the `2024` while doing all this scripting. You will forget to update this every single year, I can promise you :P